flag.txt

```diff
@@ -1 +1 @@
-Wed Nov 16 15:43:07 UTC 2022
+Sat May 27 15:40:32 UTC 2023
```

Comment:
We don't need the flag.txt anymore, since we use the keep-alive action.

Reply:
Thanks @andrew2net , removed flag.txt